### PR TITLE
Header and headline

### PR DESCRIPTION
**Pull request by ZabihullahNooriWardak**
📝Accomplishment
It is my personal Portfolio project, and I have added some new features to this repository they are as following:
● Added header section to HTML skeleton.
● Added hero section (below the header)  to HTML skeleton.
● Added Style to the HTML skeleton using CSS
● Updated the REDMI.md file with latest screenshot.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body {
+  background-color: blue;
+}
```
Comment (ZabihullahNooriWardak) [inline style.css:3]: h1 tag for test

**Comment by ZabihullahNooriWardak** [comment]
i have done some work on the very first part of the portfolio project

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi  @ZabihullahNooriWardak , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project that has a couple of areas that need improvement.
- The title of your pull request lacks professionalism. It is essential to have a clear and meaningful title that accurately describes the purpose of the pull request. Consider using titles like "Create a Sidebar for Mobile Menu" or "Add Comments Section for User" that provide a concise summary of the changes made. :+1: Please change the PR title which describes what you did in this PR.

- The pull request description should be descriptive and provide context about the changes being introduced. Take the opportunity to explain the purpose, implementation details, and any relevant information that would help reviewers understand the changes better. :+1: Please change the PR description which describes what you did in this PR.

| Your | Professional |
| --- | --- |
| image | image |

- The README file, please keep in mind that only the FAQ section is optional, all the other sections are important in the README file. Kindly take a look at the original template by clicking [here](https://github.com/microverseinc/readme-template)
I marked some parts of the template you did not implement yet. 
image




Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  font-weight: bold;
+}
+
+p {
+  color: white;
+  margin-left: 2rem;
+  font-size: 2rem;
+  line-height: 2.8rem;
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+}
```
Comment (shella12) [inline style.css:61]: - [ ] Here too the `font` defined for the template for the paragraph is `Poppins` so please make sure to change that as well.

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @ZabihullahNooriWardak, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- Fair attempt at design implementation ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+.toolbar {
+  background-color: rgb(60, 58, 57);
+  padding-top: 2rem;
+  display: flex;
+  margin-right: 2rem;
+  margin-left: 2rem;
+  font-size: 2rem;
+  justify-content: space-between;
+  align-items: baseline;
+}
```
Comment (aamirkhan2478) [inline style.css:38]: - [ ] As mentioned by the previous reviewer :point_up_2: Please use the same colors as mentioned in the Figma design but still you are not using the same color of Figma design so kindly replace `background-color: rgb(60, 58, 57);` to `background-color: #3C3A39;` Also please use `logo` and `handburger menu` color `#FFFFFF` :slightly_smiling_face: 
- [ ] As mentioned by the previous reviewer :point_up_2: also please make sure that the section has the margins (top, bottom, left, right) defined in the design but still the height of your navbar is not the same as the Figma design so kindly fix this issue :+1: check below :point_down: the comparison between your design and Figma's design :slightly_smiling_face: 

**Your Design**
Screenshot from 2023-06-02 14-59-38

**Figma Design**
Screenshot from 2023-06-02 14-59-56

File: style.css [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+h1 {
+  color: rgb(255, 107, 0);
+  font-size: 3rem;
+  margin-left: 2rem;
+  margin-top: 0;
+  margin-bottom: 0;
+  font-family: 'Crete Round', serif;
+}
```
Comment (aamirkhan2478) [inline style.css:52]: - [ ] As mentioned by the previous reviewer :point_up_2: please use color `#FF6B00` to match with the Figma design but still you didn't use same color suggested by the previous reviewer so kindly replace `color: rgb(255, 107, 0);` with `color: #FF6B00;` :+1: check below :point_down: the comparison between your design and Figma's design :slightly_smiling_face: 

**Your Design**
Screenshot from 2023-06-02 15-19-52

**Figma Design**
Screenshot from 2023-06-02 15-20-05

- [ ] Also please use same `font size` as mentioned in Figma design :+1:

File: style.css [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+h4 {
+  color: rgb(255, 245, 225);
+  margin-top: 1.7rem;
+  margin-bottom: 0.3rem;
+  margin-left: 2rem;
+  font-size: 2.4rem;
+  font-family: 'Roboto', sans-serif;
+  font-weight: bold;
+}
```
Comment (aamirkhan2478) [inline style.css:62]: - [ ] As mentioned by the previous reviewer :point_up_2: please use the same color as mentioned by the Figma design so kindly replace your color with `#FFF5E1` :+1:

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @ZabihullahNooriWardak , 



Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters passed
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @ZabihullahNooriWardak , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] No linter errors :white_check_mark: 
- [x] Good design implementation :white_check_mark: 
- [x] Pull request has a descriptive summary and title :white_check_mark: 
- [x]  Repository has a professional readme file :white_check_mark:  
- [x] Implemented changes from the previous reviewer :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**